LPS-60210 Apply new styles to empty result message

ealonso opened the pull request.

ealonso committed:
LPS-60210 Apply new styles to empty result message

liferay-continuous-integration commented:
<h1>Some tests FAILED!</h1><p>Build Time: 34 minutes 50 seconds</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br />Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/53b2a8f46029e43d792ba8e1c5e30eba105b63a4">53b2a8f46029e43d792ba8e1c5e30eba105b63a4</a></p><h4>Job Summary:</h4><ul><li><strike><strong><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest(master)/1589/">test-portal-acceptance-pullrequest(master)</a></strong></strike></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/13085/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat-hypersonic/0</a></li><li><strike><strong><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/4732/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat-mysql/0</a></strong></strike></li><li><strike><strong><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/6274/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat-mysql/1</a></strong></strike></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/13001/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat-mysql/2</a></li><li><strike><strong><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/4538/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat-mysql/3</a></strong></strike></li><li><strike><strong><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/13081/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat-mysql/4</a></strong></strike></li><li><strike><strong><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/4506/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat-mysql/5</a></strong></strike></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/6053/">test-portal-acceptance-pullrequest-batch(master)/portal-web</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-source(master)/277/">test-portal-acceptance-pullrequest-source(master)</a></li></ul><h5>For more details click <a href="https://test-1-2.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/1589/jenkins-report.html">here</a>.</h5><hr /><h4>Failed Jobs:</h4><ol><li><h5><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest(master)/1589/">test-portal-acceptance-pullrequest(master)</a></h5><h6>Job Results:</h6><p>4 Jobs Passed.<br />6 Jobs Failed.</p><pre>Completed with the status of FAILURE.</pre></li><li><h5 job-result="UNSTABLE"><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/4732/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat-mysql/0</a></h5><h6>Job Results:</h6><p>22 Tests Passed.<br />1 Test Failed.</p><ol><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=PGBookmarks%23DeleteFolder,label_exp=!master/4732//testReport/com.liferay.poshi.runner/PoshiRunner/test_PGBookmarks_DeleteFolder_">PoshiRunner.test[PGBookmarks#DeleteFolder]</a> - <a href="https://test-1-2.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest%28master%29/builds/1589/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=PGBookmarks%23DeleteFolder,label_exp=!master/functional-tomcat-mysql/0/index.html">Poshi Report</a> - <a href="https://test-1-2.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest%28master%29/builds/1589/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=PGBookmarks%23DeleteFolder,label_exp=!master/functional-tomcat-mysql/0/summary.html">Poshi Summary</a> - <a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=PGBookmarks%23DeleteFolder,label_exp=!master/4732//console">Console Output</a></li></ol></li><li><h5 job-result="UNSTABLE"><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/6274/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat-mysql/1</a></h5><h6>Job Results:</h6><p>22 Tests Passed.<br />1 Test Failed.</p><ol><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=CPSites%23DeleteSite,label_exp=!master/6274//testReport/com.liferay.poshi.runner/PoshiRunner/test_CPSites_DeleteSite_">PoshiRunner.test[CPSites#DeleteSite]</a> - <a href="https://test-1-2.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest%28master%29/builds/1589/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=CPSites%23DeleteSite,label_exp=!master/functional-tomcat-mysql/1/index.html">Poshi Report</a> - <a href="https://test-1-2.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest%28master%29/builds/1589/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=CPSites%23DeleteSite,label_exp=!master/functional-tomcat-mysql/1/summary.html">Poshi Summary</a> - <a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=CPSites%23DeleteSite,label_exp=!master/6274//console">Console Output</a></li></ol></li><li><h5 job-result="UNSTABLE"><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/4538/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat-mysql/3</a></h5><h6>Job Results:</h6><p>20 Tests Passed.<br />2 Tests Failed.</p><ol><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=BlogsUsecase%23DeleteBlogsEntry,label_exp=!master/4538//testReport/com.liferay.poshi.runner/PoshiRunner/test_BlogsUsecase_DeleteBlogsEntry_">PoshiRunner.test[BlogsUsecase#DeleteBlogsEntry]</a> - <a href="https://test-1-2.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest%28master%29/builds/1589/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=BlogsUsecase%23DeleteBlogsEntry,label_exp=!master/functional-tomcat-mysql/3/index.html">Poshi Report</a> - <a href="https://test-1-2.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest%28master%29/builds/1589/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=BlogsUsecase%23DeleteBlogsEntry,label_exp=!master/functional-tomcat-mysql/3/summary.html">Poshi Summary</a> - <a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=BlogsUsecase%23DeleteBlogsEntry,label_exp=!master/4538//console">Console Output</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=CPSitememberships%23DeleteSiteTeam,label_exp=!master/4538//testReport/com.liferay.poshi.runner/PoshiRunner/test_CPSitememberships_DeleteSiteTeam_">PoshiRunner.test[CPSitememberships#DeleteSiteTeam]</a> - <a href="https://test-1-2.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest%28master%29/builds/1589/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=CPSitememberships%23DeleteSiteTeam,label_exp=!master/functional-tomcat-mysql/3/index.html">Poshi Report</a> - <a href="https://test-1-2.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest%28master%29/builds/1589/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=CPSitememberships%23DeleteSiteTeam,label_exp=!master/functional-tomcat-mysql/3/summary.html">Poshi Summary</a> - <a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=CPSitememberships%23DeleteSiteTeam,label_exp=!master/4538//console">Console Output</a></li></ol></li><li><h5 job-result="UNSTABLE"><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/13081/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat-mysql/4</a></h5><h6>Job Results:</h6><p>21 Tests Passed.<br />1 Test Failed.</p><ol><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=RecyclebinWebcontent%23RestoreWebcontent,label_exp=!master/13081//testReport/com.liferay.poshi.runner/PoshiRunner/test_RecyclebinWebcontent_RestoreWebcontent_">PoshiRunner.test[RecyclebinWebcontent#RestoreWebcontent]</a> - <a href="https://test-1-2.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest%28master%29/builds/1589/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=RecyclebinWebcontent%23RestoreWebcontent,label_exp=!master/functional-tomcat-mysql/4/index.html">Poshi Report</a> - <a href="https://test-1-2.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest%28master%29/builds/1589/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=RecyclebinWebcontent%23RestoreWebcontent,label_exp=!master/functional-tomcat-mysql/4/summary.html">Poshi Summary</a> - <a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=RecyclebinWebcontent%23RestoreWebcontent,label_exp=!master/13081//console">Console Output</a></li></ol></li><li>...</li></ol>

ealonso commented:
Hey @evannagayama,

Could you take a look into this poshi errors? We added a new class on the empty result message and we changed the "<div>" for a "<p>"

Thanks!!!

brianchandotcom commented:
@evannagayama please resend to me directly with fixed poshi tests. Thx.
